code/pilotfile/plr.cpp

```diff
@@ -797,7 +799,8 @@ bool pilotfile::load_player(const char* callsign, player* _p, bool force_binary)
 
 	mprintf(("PLR => Loading '%s' with version %d...\n", filename.c_str(), version));
 
-	plr_reset_data();
+	//1 resets everything, 0 sets up file verify.
```

Comment:
Since the function takes a boolean argument, rather than an integer, you should use true and false instead of 1 and 0.

Reply:
Got it, wasn't sure what was acceptable.  Fixes on the way.